en/README.md

```diff
@@ -0,0 +1,1138 @@
+|[putEvents](#user-content-puteventsmethod)|Receive events view from HAP and update it.|method|O|
+|[putHostParents](#user-content-puthostparentsmethod)|Receive host parent relations view from HAP and update it.|method|O|
+|[putArmInfo](#user-content-putarminfomethod)|Receive connecting status of HAP and update it.|method|M|
+
+### HAP procedures
+
+|name|desscription|type|M/O|
+|:-------------|:---|:-----|:-:|
+|[exchangeProfile](#user-content-exchangeprofilemethod)|Receive view of procedures that is implemented in Hatohol server and Hatohol server name, after that return procedures view that is implemented in yourself and yourself name.|method|M|
+|[fetchItems](#user-content-fetchitemsmethod)|Accept request to get item from Hatohol server.|method|O|
+|[fetchHistory](#user-content-fetchhistorymethod)|Accept request to get history from Hatohol server.|method|O|
+|[fetchTriggers](#user-content-fetchtriggersmethod)|Accept request to get trigger from Hatohol server.|method|O|
+|[fetchEvents](#user-content-fetcheventsmethod)|Accept request to get event from Hatohol server.|method|O|
+|[updateMonitoringServerInfo](#user-content-updatemonitoringserverinfomethod)|Receive connecting information with Hatohol server and polling interval seconds as notification.|method|M|
+
+### exchangeProfile(method)
+
+ This procedure is had Hatohol server and HAP in common. It uses when start and restart mainly. Please look at the [About start up oetation](#user-content-about-start-up-content).
+
+***Request(params)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|procedures|String255 array|M|-|Usual procedures list of sender.|
+|name      |String255    |M|-|Process name of sender. It is used in the log message and etc.|
+
+```json
+{
+  "id": 1,
+  "params": {
+    "name": "exampleName",
+    "procedures": [
+      "getMonitoringServerInfo",
+      "getLastInfo",
+      "putItems",
+      "putArmInfo",
+      "fetchItems"
+    ]
+  },
+  "method": "exchangeProfile",
+  "jsonrpc": "2.0"
+}
+```
+
+***Result(result)***
+
+|Object name|Type |M/O|Default value|Description|
+|:-----------------|:--|:-:|:----------:|:---|
+|procedures|String255 array|M|-|Usual procedures list of destination.|
+|name      |String255    |M|-|Process name of destination. It is used in the log message and etc.|
+
+```json
+{
+  "id": 1,
+  "result": {
+    "name": "exampleName",
+    "procedures": [
+      "getMonitoringServerInfo",
+      "getLastInfo",
+      "putItems",
+      "putHistory",
+      "putHosts",
+      "putHostGroups",
+      "putHostGroupMembership",
+      "putTriggers",
+      "putEvents",
+      "putHostParent",
+      "putArmInfo"
+    ]
+  },
+  "jsonrpc": 2
+}
+```
+
+### getMonitoringServerInfo(method)
+
+To inquire myself conecting information and polling interval to Hatohol server per polling interval is the standard motion, but can inquire anytime.
```

Comment:
This procedure is mainly used to acquire Hatohol Server itself connection and polling interval information. And this procedure can call at the arbitrary point of the time. 

**Arbitrary** is more suitable here.